### PR TITLE
updates to ICS 08

### PR DESCRIPTION
**Pull request by crodriguezvega**
I have added pseudocode implementations for some of the functions and done some style changes.

### CODE REVIEW

File: spec/client/ics-008-wasm-client/README.md [hunk 1]
```diff
@@ -141,42 +137,139 @@ interface Misbehaviour {
...
 
 ### Validity predicate
 
-`Wasm Client` validity checking uses underlying `Wasm Contract`. If the provided header is valid, the client state will proceed to checking for misbehaviour and updating state.
+Wasm client validity checking uses underlying Wasm contract. If the provided client message is valid, the client state will proceed to checking for misbehaviour (call to `checkForMisbehaviour`) and updating state (call to either `updateStateOnMisbehaviour` or `updateState` depending on whether misbehaviour was detected in the client message). 
 
 ```typescript
-  function verifyClientMessage(cs: ClientState, clientStore: sdk.KVStore, clientMsg: exported.ClientMessage) {}
+function verifyClientMessage(clientMsg: ClientMessage) {
+  innerPayload = Map<string, any>()
```
Comment (crodriguezvega) [inline spec/client/ics-008-wasm-client/README.md:172]: I added this pseudocode to kinda match what the implementation does (i.e. marshalling the payload as JSON). Let me know if you agree.

File: spec/client/ics-008-wasm-client/README.md [hunk 2]
```diff
@@ -262,115 +407,116 @@ The response for queries is as follows:
...
 
-#### `Contract Execute`
-Every `Wasm Contract` must support these execute messages:
+#### Contract execute
+
+Every Wasm contract must support these execute messages:
 
 ```rust
 #[cw_serde]
 pub struct VerifyMembershipMsg {
```
Comment (crodriguezvega) [inline spec/client/ics-008-wasm-client/README.md:428]: Should this match the [raw or not raw message](https://github.com/ComposableFi/centauri/blob/vmarkushin/wasm/light-clients/ics10-grandpa-cw/src/msg.rs#L150-L170)?

File: spec/client/ics-008-wasm-client/README.md [hunk 3]
```diff
@@ -119,7 +115,7 @@ interface Height {
 
 ### Headers
 
-Contents of Wasm client headers depend upon `Wasm Contract`.
+Contents of Wasm client headers depend upon Wasm contract. Binary data represented by the `data` field is opaque and only interpreted by the Wasm contract.
```
Comment (blasrodri) [inline spec/client/ics-008-wasm-client/README.md:118]: Would "contract" be the proper term? We can instead choose a more general term like WASM "program".

**Review by blasrodri — APPROVED** [review]

File: spec/client/ics-008-wasm-client/README.md (hunk 2 above)
Comment (blasrodri) [reply spec/client/ics-008-wasm-client/README.md:428]: To me it's the "raw" version. @vmarkushin what do you think?

**Pull request merged**